staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -81,6 +81,11 @@ type ListMeta struct {
 	// identical to the value in the first response, unless you have received this token from an error
 	// message.
 	Continue string `json:"continue,omitempty" protobuf:"bytes,3,opt,name=continue"`
+
+	// RemainingItemCount is the number of remaining items if the list request does not contain label
+	// selector or field selector, and the returned list is partial. Note that servers older than
+	// v1.15 do not set this field.
+	RemainingItemCount int64 `json:"remainingItemCount,omitempty" protobuf:"bytes,4,opt,name=remainingItemCount"`
```

Comment:
I recommend making it optional, so that at least in the label/field selector case, it won't contain a misleading 0.

I guess it is already omitempty but I would also make it explicitly optional.

Reply:
Added the `// optional` comment.